openqa-maintenance.py

```diff
@@ -88,6 +90,42 @@ def settings(self, src_prj, dst_prj, packages, req=None):
         return self._settings.copy()
 
 
+class SUSEUpdate(Update):
+    def __init__(self, settings):
+        Update.__init__(self, settings)
```

Comment:
Or maybe is better to skip entirely `__init__` definition, by default will use the parent contructor